crowdfunding_compassion/templates/crowdfunding_components.xml

```diff
@@ -185,55 +185,10 @@
             </div>
         </template>
 
-         <template id="project_card" name="Project card">
```

Comment:
Why did you remove this template? It's used

Reply:
It was not removed but moved to the templates folder

Reply:
You probably forgot to commit the new file because here in the diff I only see the supression. Please check again

Reply:
Sorry I forgot that I renamed it to crowdfunding_project_card, in crowdfunding_components.xml

Reply:
in homepage there was a call to project_card, so I changed it

Reply:
Ok, good :+1: